tools/jslint.js

```diff
@@ -215,10 +215,10 @@ if (cluster.isMaster) {
     const elapsed = process.hrtime(startTime)[0];
     const mins = padString(Math.floor(elapsed / 60), 2, '0');
     const secs = padString(elapsed % 60, 2, '0');
-    const passed = padString(successes, 6, ' ');
-    const failed = padString(failures, 6, ' ');
+    const passed = padString(successes, 4, ' ');
+    const failed = padString(failures, 4, ' ');
```

Comment:
The padding is there to prevent the statistical info portion of the line from changing its width during file processing. Otherwise we could just never pad the numbers with spaces and just always have the width automatically size itself when numbers get larger. It's purely aesthetic.